projects/net-snmp/snmp_parse_fuzzer.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
```

Comment:
Are you a Google employee? If not, please fix this copyright line.

Reply:
This license is needed and am collaborating with the Google folks.